rsconf/component/sirepo_jupyterhub.py

```diff
@@ -23,6 +23,15 @@ def internal_build_compile(self):
     def sirepo_config(self, sirepo):
         self.j2_ctx_pksetdefault(sirepo.j2_ctx)
         sirepo.j2_ctx.sirepo_jupyterhub.hub_ip = self.j2_ctx.sirepo_jupyterhub.hub_ip
+        self.j2_ctx.sirepo_jupyterhub.sirepo_uri = f'https://{sirepo.j2_ctx.sirepo.vhost}'
+        c = 'SirepoAuthenticator'
+        p = sirepo.j2_ctx.sirepo.feature_config.get('default_proprietary_sim_types', [])
```

Comment:
Might be good to combine this and the other union, and just have a boolean for this.

Reply:
87e5cf5